aqua/qt4-mac/Portfile

```diff
@@ -223,7 +234,7 @@ patchfiles-append   \
 # (20) Under 10.8 and 10.9: Patch to fix corelib linking
 
 platform darwin {
-    if {${MINOR} >= 8} {
+    if {${MAJOR} > 10 || ${MINOR} >= 8} {
```

Comment:
Wouldn't it be simpler to check if `${os.major} >= 12`?

Reply:
I had the same thought. However, I decided not to change the approach used by @michaelld in this PR but just minimally extend things to support macOS11. If though @michaelld agrees I will also make these changes.